www/faq.php

```diff
@@ -75,38 +75,52 @@
 
 	<tr><td class="textcontent">
 	<p>
-	You should be familiar with the <a href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/ports.html">Ports</a>
-	section of <a href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/index.html">The FreeBSD Handbook</a>.
-	Pay careful attention to the difference between a port and a package.
+	Pay careful attention to the differences between a port and a package.
+	<p>
+	Blogged by the FreeBSD Foundation in August 2024:
+	<a href="https://freebsdfoundation.org/blog/freebsd-ports-and-packages-what-you-need-to-know/">
+	FreeBSD Ports and Packages: What you need to know</a>. More generally,
+	please be familiar with the <a href="https://docs.freebsd.org/en/books/handbook/ports/">
+	Ports</a> section of the <a href="https://docs.freebsd.org/en/books/handbook/">
+	FreeBSD Handbook</a>.
 
 	<h2>Definitions for the hopelessly lazy</h2>
 
 	<h3 id="port">What is a port?</h3>
 	<p>
-	If you install the port, the source will be downloaded, patched if necessary,
-	compiled and installed.  You will have a chance to set any optional configuration
-	items particular to that port. If the port is dependent upon another port,
-	that port will also be installed.  After installing, ports are identical to
-	packages.
-
+	If you install a port, the source will be downloaded, patched if 
+	necessary, compiled and installed. If the port has any default
+	configuration option that is preset by the maintainer, you can choose
```

Comment:
"Some ports have default configuration options. During the build process, you can set these various options."

There's a mix of plural and singular.

Reply:
Thanks. I was also not particularly happy with the mixture. I'm vaguely trying to convey: 

- some ports have no more than one option
- wherever there's an option, there's the maintainer-set default (true?)

– as succinctly as possible.

Reply:
Reminding myself (an example): 

- <https://www.freshports.org/editors/ghostwriter/#config> _config_ anchor for _Configuration options_
- <https://www.freshports.org/editors/ghostwriter/#options> _options_ anchor for _Options name_.

----

Options … configuration options … build options … compile-time options …

On one hand, I quite like the phrase _build options_.

----

On the other hand, let's compare with something on the poudriere side. 

[poudriere-options(8)](https://man.freebsd.org/cgi/man.cgi?query=poudriere-options&sektion=8&manpath=freebsd-ports)

> **poudriere options** – configure the options for a given port